src/util/mac.ts

```diff
@@ -9,10 +9,10 @@ export function validMacAddress(address: string): boolean {
 }
 
 export function generate(data: string | Buffer | NodeJS.TypedArray | DataView): MacAddress {
-  const sha1sum = crypto.createHash("sha1");
-  sha1sum.update(data);
-  const s = sha1sum.digest("hex");
+  const sha256sum = crypto.createHash("sha256");
```

Comment:
If I remember correctly, the MAC address (called username in HAP-NodeJS) is used as an identifier for all storage files. See  https://github.com/homebridge/HAP-NodeJS/blob/8804db59230b4fc8e2d4a2f9971ea05a3def28d2/src/lib/Accessory.ts#L1126

This will break all persistent storage of HAP-NodeJS users. The usage of sha1 ist not security relevant. So I would just leave it like that! Or what do you think? 
Otherwise we definitely need some migration in place.